core/roboconf-dm/src/main/java/net/roboconf/dm/management/api/ICommandsMangr.java

```diff
@@ -0,0 +1,111 @@
+ * the moral rights of the other copyright holders.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.roboconf.dm.management.api;
+
+import java.io.File;
+import java.io.IOException;
+
+import net.roboconf.core.model.beans.Application;
+
+/**
+ * An API to manipulate commands in the DM.
+ * <p>
+ * Commands are only used by applications.
+ * Application templates can come with predefined commands, but they will
+ * be copied in applications. Commands are stored in application directories, under the
+ * "commands" directory.
+ * </p>
+ *
+ * @author Vincent Zurczak - Linagora
+ */
+public interface ICommandsMangr {
+
+	/**
+	 * Creates a command from its instructions.
+	 * @param app the associated application
+	 * @param commandName the command name (must be unique)
+	 * @param commandText the instructions contained in the command (must be valid)
+	 * @throws IOException if something went wrong
+	 * @see #validate(String)
+	 */
+	void createCommand( Application app, String commandName, String commandText ) throws IOException;
+
+
+	/**
+	 * Creates a command from a command file.
+	 * @param app the associated application
+	 * @param commandName the command name (must be unique)
+	 * @param commandFile a file with command instructions (that must be valid)
+	 * @throws IOException if something went wrong
+	 * @see #validate(String)
+	 */
+	void createCommand( Application app, String commandName, File commandFile ) throws IOException;
+
+
+	/**
+	 * Deletes a command.
+	 * @param app the associated application
+	 * @param commandName the command name
+	 * @throws IOException if something went wrong
+	 */
+	void deleteCommand( Application app, String commandName ) throws IOException;
+
+
+	/**
+	 * Updates the instructions of a command.
+	 * @param app the associated application
+	 * @param commandName the command name
+	 * @param commandText the new command instructions (must be valid)
+	 * @throws IOException if something went wrong
+	 */
+	void updateCommand( Application app, String commandName, String commandText ) throws IOException;
+
+
+	/**
+	 * Gets the instructions contained by a command.
+	 * @param app the associated application
+	 * @param commandName the command name
+	 * @return the commands content (never null)
+	 */
+	String getCommandInstructions( Application app, String commandName );
+
+
+	/**
+	 * Validates the syntax of command instructions
```

Comment:
Can you add a dot at the end of this line? It breaks the build.
Our Checkstyle configuration verifies this kind of things.

Please, do not commit any other modification.